### PR TITLE
Feat: Added link to AnitaB.Org in contact menu

### LINKED ISSUE #98: Feat: Add link to AnitaB.Org Open Source contact page on sub menu `Contact` under `Home`
## Description
As a public user,
I need see a contact page for AnitaB.Org Open Source,
so that I know who to contact if I want to ask something in regard to BridgeInTech.

## Mocks
NA

### Update [Required]
- [ ] The `Contact` sub menu under `Home` Navbar must redirect user to [AnitaB.Org Contact Page](https://anitab.org/contact/)

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

**Additional Information**
To do this task, you do not need to setup BridgeInTech Backend environment (if you haven't already done so), since there is no login required.

### PR DESCRIPTION
**Pull request by Harshita-Kanal**
### Description

Fixes #98 

### Type of Change:

- Code
- Quality Assurance
- Outreach
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Tested Locally

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by gaurivn — DISMISSED** [review]
Good work :tada: thank you

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@Harshita-Kanal , can you please make the link open to a new tab? This is so if the user still have our website open in a separate tab, they are more likely continue exploring BridgeInTech web and not distracted to explore AnitaB.Org site (so here we are aiming for higher retention 😉)

File: src/Navigation.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default function Navigation() {
                                     <Nav>
                                         <Nav.Link as={Link} to="/about">About</Nav.Link>
                                         <Nav.Link as={Link} to="/benefit">Benefit</Nav.Link>
-                                        <Nav.Link as={Link} to="/contact">Contact</Nav.Link>
+                                        <Nav.Link href="https://anitab.org/contact/">Contact</Nav.Link>
```
Comment (Kajol-Kumari) [inline src/Navigation.js]: Please use something like `target="_blank"` attribute here so that we don't loose the opened page of the site.

**Comment by Harshita-Kanal** [comment]
I made the changes, I don't know why is it showing the dismissed review comment, I just pushed a commit. Thanks.

**Comment by mtreacy002** [comment]
> I made the changes, I don't know why is it showing the dismissed review comment, I just pushed a commit. Thanks.

This is normal, @Harshita-Kanal , since you've pushed a new update, the approval given prior to the new push automatically got dismissed. You'll see this often, so, don't be alarmed 😉.

**Review by mtreacy002 — APPROVED** [review]
This LGTM, @Harshita-Kanal. Great job on your First-Timers-Only issue. 🎉🎉
Thank you again for contributing to the project. 👍
Don't forget to join the [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech) zulip channel to keep up with the project progress and discussions 😉

**Comment by mtreacy002** [comment]
@Harshita-Kanal ,  can you please add `Feat: ...` (aka Feature) in front of the title of this PR? Just so when we browse through thee list of PR we know what type of PR this is. 
Also just few little things I need to mention for you to apply in the next contribution. 
* Please make sure you follow the [Commit Message Style Guide](https://github.com/anitab-org/bridge-in-tech-web/wiki/Commit-Message-Style-Guide) that is mentioned in the [Getting Started wiki page](https://github.com/anitab-org/bridge-in-tech-web/wiki/Getting-Started) under section `Start Working` so to maintain consistency across the repositories.

**Comment by Harshita-Kanal** [comment]
Sure, thank you

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:  
Thank you for your contribution @Harshita-Kanal  :)

**Pull request merged**